db/migrate/20221216154033_add_payment_retry_columns_to_invoices.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddPaymentRetryColumnsToInvoices < ActiveRecord::Migration[7.0]
+  def change
+    add_column :invoices, :payment_attempts, :integer, default: 0, null: false
```

Comment:
Rails/BulkChangeTable: You can use `change_table :invoices, bulk: true` to combine alter queries.